src/pages/experiments/[experimentId]/data-exploration/components/embedding/Embedding.jsx

```diff
@@ -212,7 +209,7 @@ const Embedding = (props) => {
         cells={convertCellsData(data)}
         mapping='PCA'
         selectedCellIds={selectedCellIds}
-        cellColors={(selectedCell) ? { ...cellColors, [selectedCell]: [0, 0, 0] } : cellColors}
+        cellColors={(selectedCell) ? new Map(Object.entries({ ...cellColors, [selectedCell]: [0, 0, 0] })) : new Map(Object.entries(cellColors))}
```

Comment:
Is this related to the bug where the coloring was glitching out?

Reply:
I have no idea why Vitessce introduced it, but now it it required to pass `Map` instead of plain `object`.